DEPRECATED.md

```diff
@@ -0,0 +1,10 @@
+# DEPRECATED
+
+As of release 1.3.0, Envoy will follow a
+[Breaking Change Policy](https://github.com/lyft/envoy/blob/master//CONTRIBUTING.md#breaking-change-policy).
+
+The following features have been DEPRECATED and will be removed in the specified release cycle.
+
+* Version 1.4.0
+  * Config option `statsd_local_udp_port` has been deprecated and has been replaced with
```

Comment:
thanks @hennna I'll update this with https://github.com/lyft/envoy/commit/3a466c3506a97df38a9efb15dff85e2a7e75b556
once you merge it